### PR TITLE
Extract scalar reading tests into separate file

### PR DESCRIPTION
**Pull request by OptimumCode**
Related to #27

### CODE REVIEW

File: src/commonTest/kotlin/com/charleskorn/kaml/YamlTestUtil.kt
Comment (charleskorn) [inline src/commonTest/kotlin/com/charleskorn/kaml/YamlTestUtil.kt:1]: Is there a better name for this file? Perhaps `TestObjects` or something like that?

**Review by charleskorn — COMMENTED** [review]
Thanks for working on this!

File: src/commonTest/kotlin/com/charleskorn/kaml/YamlTestUtil.kt
Comment (OptimumCode) [reply src/commonTest/kotlin/com/charleskorn/kaml/YamlTestUtil.kt:1]: Hi, @charleskorn. How about `TestSerializers` or `LocationAwaredSerializers`?

Comment (charleskorn) [reply src/commonTest/kotlin/com/charleskorn/kaml/YamlTestUtil.kt:1]: `TestSerializer` sounds good to me.

Comment (OptimumCode) [reply src/commonTest/kotlin/com/charleskorn/kaml/YamlTestUtil.kt:1]: Done

**Review by charleskorn — APPROVED** [review]

**Comment by charleskorn** [comment]
Thanks @OptimumCode!

**Pull request merged**